ui/imports/shared/popups/SignTransactionModal.qml

```diff
@@ -127,7 +127,7 @@ StatusModal {
                     id: selectRecipient
                     visible: false
                     accounts: root.store.walletModelInst.accountsView.accounts
```

Comment:
Could you also replace that by account from refactored wallet? I hope it works :crossed_fingers:

Reply:
Done :)